src/main.cc

```diff
@@ -1010,7 +1010,8 @@ mainReconfigureFinish(void *)
 
     neighbors_init();
 
-    storeDirOpenSwapLogs();
+    if (!IamCoordinatorProcess())
```

Comment:
Please move the new process role check from main.cc into Fs::Ufs::UFSSwapDir::openLog() in src/fs/ufs/UFSSwapDir.cc, placing it before all the assertions there.

Reply:
Done